We do already provide a desktop GL build of qt5,
so assuming that AArch64 is a OpenGL ES platform
is wrong for us

--- a/libs/ui/CMakeLists.txt	2021-08-03 10:57:54.000000000 +0200
+++ -	2021-08-09 13:40:50.981311691 +0200
@@ -7,12 +7,6 @@
     ${OCIO_INCLUDE_DIR}
 )
 
-if (ANDROID OR (CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64.*|AARCH64.*|arm64.*|ARM64.*)" AND NOT APPLE))
-    add_definitions(-DQT_OPENGL_ES_3)
-    add_definitions(-DHAS_ONLY_OPENGL_ES)
-    include_directories (${Qt5AndroidExtras_INCLUDE_DIRS})
-endif()
-
 add_subdirectory( tests )
 
 if (APPLE)
